docs/setup/native.md

```diff
@@ -86,19 +86,19 @@ Specific notes for our most common native installation platforms are in this sec
 ### OSX
 
 All of the OSX instructions assume `homebrew` is your [package manager](https://brew.sh/)
-
-1. Postgresql 11 can be installed
-   `brew install postgresql@11`
 1. Install Redis
    - `brew install redis`
-1. Install PostGIS 2.5
-   - `brew services start postgresql@11`
-   - Install the `pex` manager to add your Postgresql 11 extensions from [here](https://github.com/petere/pex#installation)
-   - Install the `postgis` extension along with a number of patches using the instructions summarized [here](https://gist.github.com/skissane/0487c097872a7f6d0dcc9bcd120c2ccd):
-   - ```bash
-      PG_CPPFLAGS='-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H -I/usr/local/include' CFLAGS='-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H -I/usr/local/include' pex install postgis
-
-   It may be MUCH easier to use the Postgres.app which makes installing the correct combination of Postgresql and PostGIS versions.
+1. Install Postgresql & PostGIS
+   1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+   1. Alternatively Postgresql 11 & PostGIS 2.5 can be installed with homebrew
+      - `brew install postgresql@11`
+      - `brew services start postgresql@11`
+      - Install the `pex` manager to add your Postgresql 11 extensions from [here](https://github.com/petere/pex#installation)
+      - Install the `postgis` extension along with a number of patches using the instructions summarized [here](https://gist.github.com/skissane/0487c097872a7f6d0dcc9bcd120c2ccd):
+      - ```bash
+         PG_CPPFLAGS='-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H -I/usr/local/include' CFLAGS='-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H -I/usr/local/include' pex install postgis
```

Comment:
As a newcomer to `vets-api` when I read something like this:

> It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) blah blah blah
> Alternatively Postgresql 11 & PostGIS 2.5 can be installed blah blah blah

I really have no idea which route to chose. I'd rather be told exactly what to do. Alternatives can be decided later if & when the person needs and/or understands them.

That would mean trimming down the instructions to just include one path or the other, and then having a link to "further reading on alternatives" or something.

Reply:
Also, I understand that you didn't write this section 😄 , you just moved it.